orchagent/port/porthlpr.cpp

```diff
@@ -749,6 +761,60 @@ bool PortHelper::parsePortDescription(PortConfig &port, const std::string &field
     return true;
 }
 
+bool PortHelper::parsePortLinkEventDampingAlgorithm(PortConfig &port, const std::string &field, const std::string &value) const
```

Comment:
Can you please confirm that this is applied when user dynamically changes the link damping configs and not just during initialization?

Reply:
Yes, that is correct. Link event damping algorithm and config can be applied at runtime by changing the entries in CONFIG DB.